### PR TITLE
Release notes and documentation for FAPI 2

### LINKED ISSUE #21945: Release notes for FAPI 2
### Description

Once all the tasks from epic https://github.com/keycloak/keycloak/issues/20708 are completed and we have FAPI 2 testsuite passing.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
Closes #21945

### CODE REVIEW

**Comment by mposolda** [comment]
@tnorimat @andymunro Can you please review this?

File: docs/documentation/release_notes/topics/23_0_0.adoc [hunk 1]
```diff
@@ -4,4 +4,9 @@ Message properties files for themes are now read in UTF-8 encoding, with an auto
 
 See the migration guide for more details.
 
+= FAPI 2 drafts support
+
+Keycloak has new client profiles `fapi-2-security-profile` and `fapi-2-message-signing`, which ensure Keycloak to enforce compliance with
```
Comment (andymunro) [inline docs/documentation/release_notes/topics/23_0_0.adoc:9]:
```suggestion
Keycloak has new client profiles `fapi-2-security-profile` and `fapi-2-message-signing`, which ensure Keycloak enforces compliance with
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good. Just one minor change.

**Comment by mposolda** [comment]
@andymunro Thank you

**Review by tnorimat — APPROVED** [review]
LGTM.

**Comment by mposolda** [comment]
@tnorimat Thanks!

**Pull request merged**